back/engines/commercial/user_custom_fields/app/controllers/user_custom_fields/web_api/v1/ref_distributions_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module UserCustomFields
+  module WebApi::V1
+    class RefDistributionsController < ::ApplicationController
+      def show
+        render json: serialize(reference_distribution)
+      end
+
+      def create
+        ref_distribution = authorize(Representativeness::RefDistribution.new(create_params))
+
+        Representativeness::RefDistribution.transaction do
+          remove_reference_distribution_if_any(params[:custom_field_id])
+          ref_distribution.save!
+        end
+
+        side_fx.after_create(ref_distribution, current_user)
```

Comment:
`before_create` is missing from this method.

Reply:
I don't really see the issue here since there are no `before_create` effects in this resource. Of course, we might add some in the future, but I'd rather not anticipate and clutter the code with method calls that do nothing.

Reply:
I see that `Representativeness::SideFxRefDistributionService` inherits from `BaseSideFxService`, which does not implement `before_create`, so my comment was not valid. I expected `before_create` to be in the superclass, because it is present in similar classes, like `SideFxCommentService`. Apparently not all classes implement the side effects framework correctly.

So I will resolve this conversation.